apps/rewards/app/components/App/App.js

```diff
@@ -27,7 +27,7 @@ class App extends React.Component {
     displayMenuButton: PropTypes.bool.isRequired,
     fromPath: PropTypes.exact({
       selected: PropTypes.number,
-      rewardId: PropTypes.number
+      rewardId: PropTypes.string
```

Comment:
I see what you mean, we need to really change this type, but it falls out of scope since it is coming from the store this way.